src/Pos/Launcher/Scenario.hs

```diff
@@ -111,6 +112,6 @@ initLrc = do
 initUSMemState :: WorkMode ssc m => m ()
 initUSMemState = do
     tip <- GS.getTip
-    tvar <- mvState <$> askUSMemVar
+    tvar <- mvState <$> askContext @UpdateContext ucMemState
```

Comment:
Actually `@UpdateContext` isn't needed here – it's only for clarity. I don't want to add `askUSContext` and so on because it would introduce O(n) new names and I'm scared of O(n). However, if you can think of a shorter name for `askContext`, that'll be good.